alg/teca_tc_wind_radii_stats.py

```diff
@@ -158,7 +158,7 @@ def execute(self, port, data_in, req):
             month[-1],day[-1],year[-1]), fontweight='bold', fontsize=12)
         plt.subplots_adjust(hspace=0.35, wspace=0.35, top=0.90)
 
-        plt.savefig(self.output_prefix + 'wind_radii_stats.png')
+        plt.savefig(self.output_prefix + '_wind_radii_stats.png')
```

Comment:
please do not make this change. having no separator was intentional. for instance if the user specifies '/home/abdel/' as the prefix this change would result in files named with a leading `_`. applies to other scripts where `_` was added.